runtime/handler_test.go

```diff
@@ -128,7 +129,7 @@ func TestForwardResponseStream(t *testing.T) {
 						t.Errorf("marshaler.Marshal() failed %v", err)
 					}
 					errBytes := body[len(want):]
-					if string(errBytes) != string(b) {
+					if string(errBytes) != string(b) + string(delimiter) {
```

Comment:
This should be
```suggestion
					if string(errBytes) != string(b)+string(delimiter) {
```
to satisfy the linter